src/eduid_common/session/redis_session.py

```diff
@@ -309,7 +329,7 @@ def encrypt_data(self, data_dict: Mapping[str, Any]) -> str:
                 self.secret_box.encrypt(data_json.encode('ascii'), nonce, encoder=nacl.encoding.Base64Encoder)
             ).decode('ascii')
         }
-        return json.dumps(versioned)
+        return json.dumps(versioned).encode('utf-8')
```

Comment:
json.dumps has `ensure_ascii=True` as default so should that be False maybe? What are we trying to achieve here?